src/UI/EmailHTMLTooltip.js

```diff
@@ -1,6 +1,9 @@
 import { formatDuckAddress, escapeXML } from '../autofill-utils.js'
 import HTMLTooltip from './HTMLTooltip.js'
 
+/** A zero-width space literal. Some editors render it as a regular space. */
+const ZWSP = '​';
+
```

Comment:
given how this looks in GH and certain editors, what about using a unicode escape sequence instead? I think it should work, but I havn't checked :)

Reply:
Can-do!